app/helpers/application_helper.rb

```diff
@@ -518,7 +518,7 @@ def user_set?(field)
   end
 
   def notifications
-    content_tag :div, :id => 'notifications', :'data-flash' => flash.to_json.html_safe do
+    content_tag :div, :id => 'notifications', :'data-flash' => flash.to_a.select {|flash| ['notice', 'success', 'info'].include?(flash.first) }.to_json.html_safe do
```

Comment:
The toast notifications js assumes, that the flash hash only contains `['notice', 'success', 'info']` keys.